src/common/freeze/deepfreeze.ts

```diff
@@ -0,0 +1,21 @@
+
+export const deepFreezeSkipDate = (o) => {
```

Comment:
It seems secs it's the way to go for reducers:

https://github.com/rt2zz/redux-persist/issues/82